controllers/storageautoscaler/storageautoscaler_controller.go

```diff
@@ -0,0 +1,293 @@
+package storageautoscaler
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	errs "errors"
```

Comment:
Why not use `errors` directly in the code here, instead of abbreviating `errs`?

Reply:
as using two different erorrs import
`	errs "errors" `
`	"k8s.io/apimachinery/pkg/api/errors"`

Reply:
better to give more pref to stdlib and import as
```
"errors"
kerrors "k8s.io/apimachinery/pkg/api/errors"
```
`k8serrors` would be more discoverable.